introduction.md

```diff
@@ -1,2 +1,8 @@
 I'm Fabien, the worst samurai the world has seen
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?